### PR TITLE
Add dnn-model-services integration test

### PR DESCRIPTION
**Pull request by stellarspot**
An integration test which runs dnn-model-services sample with MPE contract

### CODE REVIEW

File: utils.go [hunk 1]
```diff
@@ -217,3 +217,47 @@ func checkWithTimeout(timeout time.Duration,
 func toString(value int) string {
 	return strconv.Itoa(value)
 }
+
+var outputFile string
+var outputContainsStrings []string
+var outputSkipErrors []string
```
Comment (vsbogd) [inline utils.go]: Why this variables a global? Should they be a parameters of the checkFileContainsStrings

File: features/publish_example_service.feature [hunk 2]
```diff
@@ -1,22 +1,40 @@
...
+		When  dnn-model mpe service is registered
+			| name                | endpoint       | group  |
+			| ExampleOrganization | http://localhost:8090 | group1 |
+		When  dnn-model service snet-daemon config file is created
+            | daemon port | ethereum endpoint port | passthrough endpoint port | price |
+            | 8090        | 8545                   | 7003                      | 10    |
+		When dnn-model service is running
+		When dnn-model open the payment channel
+		When dnn-model compile protobuf
+		Then dnn-model make a call using stateless logic
```
[the remark this answers is not in the document]
Comment (stellarspot) [reply features/publish_example_service.feature]: Updated.

File: utils.go (hunk 1 above)
Comment (stellarspot) [reply utils.go]: Moved fields to a separate struct and pass them as a function argument.

**Comment by stellarspot** [comment]
Found an issue that snet-cli does not fails if the service returns an error: https://github.com/singnet/snet-cli/issues/102

File: features/publish_example_service.feature [hunk 3]
```diff
@@ -1,22 +1,43 @@
...
+			| ExampleOrganization | http://localhost:8090 | group1 |
+		When  dnn-model service snet-daemon config file is created
+            | daemon port | ethereum endpoint port | passthrough endpoint port | price |
+            | 8090        | 8545                   | 7003                      | 10    |
+		When dnn-model service is running
+		When dnn-model open the payment channel
+		When dnn-model compile protobuf
+		Then dnn-model make a call using payment channel
+		Then dnn-model claim channel by treasurer server
+			| daemon port | ethereum endpoint port | passthrough endpoint port |
```
Comment (vsbogd) [inline features/publish_example_service.feature]: ```daemon port``` and ```passthrough endpoint port``` should not be required for channel claiming.

Comment (vsbogd) [reply features/publish_example_service.feature]: As discussed will be raised as separate PR

**Pull request merged**